keyboards/redox/keymaps/jeherve/config.h

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2018 Jeremy Herve <jeremy@jeremy.hu>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
```

Comment:
Could you remove this ifndef line, the define below it, and replace it with `#pragma once`,  and remove the endif at the end of the file? 

It accomplishes the same thing, but it's simpler, less prone to issues, and cleaner.